src/blocks/extensions/condition-block/index.js

```diff
@@ -123,4 +118,37 @@ const UserConditionOptions = ( props ) => {
     );
 }
 
-export default UserConditionOptions;
\ No newline at end of file
+const AdvancedControlsBlock = wp.compose.createHigherOrderComponent((BlockEdit) => {
+
+	return (props) => {
+		
+		const { Fragment } = wp.element;
+		
+		const { InspectorAdvancedControls } = wp.blockEditor;
+		
+		const { isSelected } = props;
+		
+		const blockName = props.name;
+		
+		const blockType = ['uagb/buttons-child','uagb/faq-child', 'uagb/icon-list-child', 'uagb/social-share-child', 'uagb/restaurant-menu-child', 'wpforms/form-selector','formidable/simple-form','formidable/calculator','llms/lesson-navigation','llms/pricing-table','llms/course-syllabus','llms/instructors','core/archives','core/calendar','core/latest-comments','core/tag-cloud','core/rss','real-media-library/gallery'];
+		return (
+			<Fragment>
+				<BlockEdit {...props} />
+				{isSelected && ! blockType.includes(blockName) &&
+					<InspectorAdvancedControls>
+						<p className="components-base-control__help">{ __( "Below setting will only take effect once you are on the live page, and not while you're editing.", 'ultimate-addons-for-gutenberg' ) }</p> 
+						{ UserConditionOptions( props ) }						
+					</InspectorAdvancedControls>
+				}
+			</Fragment>
+		);
+	};
+}, 'AdvancedControlsBlock');
+
+if( '1' === enableConditions ){
+	wp.hooks.addFilter(
+		'editor.BlockEdit',
+		'uagb/advanced-control-block',
+		AdvancedControlsBlock,
+	);
+}
```

Comment:
```suggestion
}

```

@vrundakansara Setup editor [Editorconfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) which takes care of this automatically.

Reply:
Done